src/histogram.rs

```diff
@@ -194,25 +194,39 @@ impl Default for HistogramCore {
     }
 }
 
-/// `HistogramTimer` represents an event being timed.
+/// `HistogramTimer` represents an event being timed. When the timer goes out of
+/// scope, the duration will be observed, or call `observe_duration` to manually
+/// observe.
```

Comment:
Add some notes to remind that a Timer could be observed only once (automatically or manually).